flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/StatusRecorder.java

```diff
@@ -196,6 +197,7 @@ public Instant getTimestamp() {
                                     listener.onSessionJobStatusUpdate(ctx);
                                 }
                             });
+                    ListenerUtils.logContext(ctx);
```

Comment:
Not sure why we need to add this to `ListenerUtils`. If @gyfora pointed this out in the jira then +1 to inline it here :)

Reply:
With this approach we have a nice and aligned table on these logs, when I grep it is easier to grep on ListenerUtils.
```
2022-09-06 13:19:47,920 o.a.f.k.o.l.ListenerUtils      [INFO ] [default.basic-example] >>> Status | Info    | UPGRADING       | The resource is being upgraded 
2022-09-06 13:19:48,030 o.a.f.k.o.l.ListenerUtils      [INFO ] [default.basic-example] >>> Event  | Info    | Submit          | Starting deployment
2022-09-06 13:19:48,672 o.a.f.k.o.l.ListenerUtils      [INFO ] [default.basic-example] >>> Status | Info    | DEPLOYED        | The resource is deployed/submitted to Kubernetes, but it’s not yet considered to be stable and might be rolled back in the future 
2022-09-06 13:19:52,700 o.a.f.k.o.l.ListenerUtils      [INFO ] [default.basic-example] >>> Status | Info    | DEPLOYED        | The resource is deployed/submitted to Kubernetes, but it’s not yet considered to be stable and might be rolled back in the future 
2022-09-06 13:20:02,857 o.a.f.k.o.l.ListenerUtils      [INFO ] [default.basic-example] >>> Event  | Info    | StatusChanged   | Job status changed from RECONCILING to RUNNING
2022-09-06 13:20:02,927 o.a.f.k.o.l.ListenerUtils      [INFO ] [default.basic-example] >>> Status | Info    | STABLE          | The resource deployment is considered to be stable and won’t be rolled back 
```

Reply:
We're not simply logging for debugging purposes, I added extra logic to reflect the content users would normally see

Reply:
This should tell a nice story to the user.

Reply:
I agree w/ you basically. If I understand correctly this is mainly for user and tracks history.
Can we rename `ListenerUtils` to something more meaningful?

Reply:
Moved the logging to a new `AuditUtils` class which makes more sense agree :)